src/main/java/at/hannibal2/skyhanni/features/rift/area/mountaintop/TimiteTracker.kt

```diff
@@ -0,0 +1,138 @@
+import at.hannibal2.skyhanni.features.rift.RiftApi
+import at.hannibal2.skyhanni.features.rift.RiftApi.motesNpcPrice
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.CollectionUtils.addSearchString
+import at.hannibal2.skyhanni.utils.ItemUtils
+import at.hannibal2.skyhanni.utils.NeuInternalName
+import at.hannibal2.skyhanni.utils.NeuInternalName.Companion.toInternalName
+import at.hannibal2.skyhanni.utils.NeuItems
+import at.hannibal2.skyhanni.utils.NeuItems.getItemStack
+import at.hannibal2.skyhanni.utils.NumberUtil.shortFormat
+import at.hannibal2.skyhanni.utils.TimeUtils.format
+import at.hannibal2.skyhanni.utils.renderables.Renderable
+import at.hannibal2.skyhanni.utils.renderables.Searchable
+import at.hannibal2.skyhanni.utils.renderables.toSearchable
+import at.hannibal2.skyhanni.utils.tracker.ItemTrackerData
+import at.hannibal2.skyhanni.utils.tracker.SkyHanniItemTracker
+import kotlin.time.Duration.Companion.seconds
+
+@SkyHanniModule
+object TimiteTracker {
+
+    private val config get() = SkyHanniMod.feature.rift.area.mountaintop.timite
+    private val HIGHLITE = "HIGHLITE".toInternalName()
+    private val TIMITE = "TIMITE".toInternalName()
+
+    class Data : ItemTrackerData() {
+        override fun resetItems() {
+            return
+        }
+
+        override fun getDescription(timesGained: Long): List<String> {
+            return emptyList()
+        }
+
+        override fun getCoinName(item: TrackedItem): String {
+            return "§5Motes"
+        }
+
+        override fun getCoinDescription(item: TrackedItem): List<String> {
+            return emptyList()
+        }
+
+        override fun getCustomPricePer(internalName: NeuInternalName): Double {
+            return internalName.getItemStack().motesNpcPrice() ?: 0.0
+        }
+
+        fun getTime(): Int {
+            this.items[TIMITE]?.let {
+                return it.totalAmount.toInt() * 2
+            }
+            return 0
+        }
+    }
+
+    private fun drawDisplay(data: Data): List<Searchable> = buildList {
+        addSearchString("§9§lTimite Tracker")
+        val profit = tracker.drawItems(data, { true }, this)
+
+        val highliteRecipe = NeuItems.getRecipes(HIGHLITE).singleOrNull()
+        if (highliteRecipe != null) {
+            var craftableAmount = 0
+
+            for (neededItem in ItemUtils.neededItems(highliteRecipe)) {
+                if (neededItem.key in validItems) {
+                    data.items[neededItem.key]?.let {
+                        val amountCanCraft = it.totalAmount.toInt() / neededItem.value
+                        if (craftableAmount == 0 || amountCanCraft < craftableAmount) {
+                            craftableAmount = amountCanCraft
+                        }
+                    }
+                }
+            }
+            val stack = HIGHLITE.getItemStack()
+            val motes = stack.motesNpcPrice()?.times(craftableAmount)?.shortFormat() ?: "0"
```

Comment:
create a new function for motesNpcPrice that accepts neu item name, so that we dont need to use item stacks here.